### ISSUE TITLE
included libraries; recent xz-utils / liblzma compromise

### ISSUE THREAD

**Issue by mralusw** [vim/vim-appimage#69]
I ran
```shell
$ GVim-v9.1.0228.glibc2.29-x86_64.AppImage &
$ ls -d /tmp/.mount*  # get auto-mounted AppDir
# /tmp/.mount_GVim-vRwbSEr

$ ldd /tmp/.mount_GVim-vRwbSEr/usr/bin/vim | grep -E 'lzma|systemd|perl|ruby|python|lua'
	liblua5.3.so.0 => /tmp/.mount_GVim-vRwbSEr/usr/bin/../lib/liblua5.3.so.0 (0x00007efc1b9f4000)
	libperl.so.5.30 => /tmp/.mount_GVim-vRwbSEr/usr/bin/../lib/libperl.so.5.30 (0x00007efc1b600000)
	libpython2.7.so.1.0 => /tmp/.mount_GVim-vRwbSEr/usr/bin/../lib/libpython2.7.so.1.0 (0x00007efc1b200000)
	libruby-2.7.so.2.7 => /tmp/.mount_GVim-vRwbSEr/usr/bin/../lib/libruby-2.7.so.2.7 (0x00007efc1ac00000)
	libsystemd.so.0 => /tmp/.mount_GVim-vRwbSEr/usr/bin/../lib/libsystemd.so.0 (0x00007efc19cc2000)
	liblzma.so.5 => /tmp/.mount_GVim-vRwbSEr/usr/bin/../lib/liblzma.so.5 (0x00007efc19c74000)

```

Two things to note here:

* The appimages *do* include `libruby`, `llibperl`, `liblua` and (interestingly) `libpython2` (but not 3) which is different from what the release notes / documentation state. I had some release note PR's coming up in any case for unrelated stuff (`AppRun.extracted`), so I ended up verifying and rewriting the docs.
* by now I suppose everyone has heard of the `xz` / `liblzma5` / `sshd` backdoor (e.g. see the [Debian fallout](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1068024)).
  * in short, Debian stable doesn't seem to be affected, and the vulnerability only seems to target Debian/Redhat-patched `sshd` anyway
  * because the `vim-appimage`'s are currently built with Ubuntu 20.04, we probably link to (and "ship") even older versions. I think `vim` can access compression algorithms (e.g. via its embedded languages) so this might still be a relevant point
  * the appimage compression algorithm (for the embedded `squashfs`) is `gzip`, not `xz` (see below)

```shell
$ GVim-v9.1.0228.glibc2.29-x86_64.AppImage --appimage-offset
193728
$ dd if=GVim-v9.1.0228.glibc2.29-x86_64.AppImage of=/tmp/x.squashfs bs=193728 skip=1
# ...
$ file /tmp/x.squashfs
/tmp/x.squashfs: Squashfs filesystem, little endian, version 4.0, zlib compressed  # ...
```

**Comment by chrisbra** [comment]
yes, so we should not be affected. So anything to do for us here?

**Comment by mralusw** [comment]
On the compromise, no, we should be more than OK (but I thought I'd check).

I'm trying to figure out what's going on with the embedded languages. We clearly ship some libraries as `ldd` shows (and unlike what the release notes state).
* Python 3 apparently does not run out of the box (`:python3` yields E319).
* Python 2 also does not work (`E887`) but it seems like it might be fixable by using the target's runtime (?)
* `Lua` seems to work and fixed at 5.3
* `Perl` at 5.30

I think the release notes are also wrong about installing those interpreters on the system &mdash; if they work at all, they will work with the builtin appimage version.

**Comment by chrisbra** [comment]
fixed by #70
